scss/media.scss

```diff
@@ -0,0 +1,428 @@
+@import "vars";
+
+@media (min-width: 1440px) {
+  .wrapper {
+    max-width: 1440px;
+    margin: auto;
+  }
+
+  .gamburgerContainer {
+    display: none;
+    position: absolute;
+  }
+
+  .logo {
+    margin: 28px 84px 0px 84px;
+    .skoutTop {
+      width: 110px;
+      height: 24px;
+    }
+  }
+
+  .containerHeaderMenu {
+    display: flex !important;
+    position: static ;
+    background-color: transparent;
+    flex-direction: row;
+    flex-basis: 78%;
```

Comment:
it is a magic number, how did you calculate this value?

Reply:
I counted by eye, done.